package.json

```diff
@@ -31,9 +30,12 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
+  "peerDependencies": {
+    "coffeeify": "^1.0.0"
```

Comment:
Ugh, yeah, that makes sense. I just removed coffeeify and I get the same thing.

I'll push a fix in a minute.